### ISSUE TITLE
Add URI Scheme to UWP for use with Retropass

### ISSUE THREAD

**Issue by danprice142** [flyinghead/flycast#668]
What should happen
A rom with per game settings should be able to be launched from Retropass using URI Scheme for UWP and when the rom is closed it should close Flycast and return to Retropass

https://github.com/retropassdev/RetroPass

Who would this benefit
Xbox users and people who are new to emulation would benefit as Retropass provides a very familiar look for xbox users and does not have any setting to mess up this way we could set up Flycast with all per game configs etc and then use Retropass as no settings can get changed/reset because of this it makes it very friendly for people new to emulation and kids.

Platform (if relevant)
UWP

Games this would be useful in
All

Other emulators or software with a similar feature
https://github.com/retropassdev/RetroPass
https://github.com/libretro/RetroArch/issues/13581
And Retrix Gold is adding support for retropass

**Comment by flyinghead** [comment]
What needs to be done in the core to support this?

A retroarch core is a dll and can't be run by itself. Only the frontend needs to support URI launch, which RA now does apparently.

**Comment by danprice142** [comment]
Yes it does and the core works with it but I was wandering if this could be added to the uwp build of the flycast

**Comment by flyinghead** [comment]
Ok, understood

**Comment by danprice142** [comment]
Started a Bounty for this Feature https://app.bountysource.com/issues/109357512-add-uri-scheme-to-uwp-for-use-with-retropass

**Comment by danprice142** [comment]
Enhancement completed with https://github.com/flyinghead/flycast/pull/2095
